internal/server/iigointernal/executive.go

```diff
@@ -180,3 +219,14 @@ func checkRuleIsValid(ruleName string, rulesCache map[string]rules.RuleMatrix) b
 func getIslandAlive() []float64 {
 	return rules.VariableMap[rules.IslandsAlive].Values
 }
+
+// incur charges in both budget and commonpool for performing an actions
+// actionID is typically the function name of the action
+// only return error if we can't withdraw from common pool
+func (e *executive) incurServiceCharge(cost shared.Resources) bool {
+	_, ok := WithdrawFromCommonPool(cost, e.gameState)
```

Comment:
I know it's a nitpick, but instead of ignoring this value, could we assign and then on line 229 subtract this value. Just to be clear to anyone reading.